auth/data-structures.md

```diff
@@ -37,10 +37,7 @@ interface RequestParams {
 ## Respond Params
 
 ```typescript
-interface RespondParams {
-  id: number;
-  signature: CacaoSignature;
-}
+type RespondParams = ResultResponse | ErrorResponse;
```

Comment:
we also need an id for error response so maybe something like?:

```suggestion
type RespondParams = (id: number, result: ResultResponse | ErrorResponse);
```

Reply:
What about using `id` param inside `ResultResponse`? Should we add `id` to `ErrorResponse` as well?

Reply:
I added id to ErrorResponse instead since ResultResponse already has an id